### DISCUSSION TITLE
Better Maps (Item)

### DISCUSSION THREAD

**Discussion by IdotIcom** [BedrockTweaks/Files#381] [Ideas]
Like this

![17177607-capturadepantalla_l](https://github.com/user-attachments/assets/fa23f198-e158-4277-810f-9352c46fe7cb)
Image from [D°Uzi Maps](https://www.planetminecraft.com/texture-pack/d-uzi-maps/)

**Comment by SquatchHunter** [comment]
interesting but technically out of scope, will be considered down the road (after addons)
